K-NN refresh blog

kolchfa-aws opened the pull request:
### Description
Refreshes an old k-NN blog
 
### Issues Resolved
Fixes #1247 

### Check List
- [x ] Commits are signed per the DCO using --signoff


By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

krisfreedain commented on _posts/2023-01-09-knn-similarity-refresh.markdown:
```diff
@@ -0,0 +1,215 @@
+excerpt: Since the OpenSearch Project introduced the k-nearest neighbor plugin in 2019, it has supported both exact and approximate k-NN search. The approximate k-NN search method is more efficient for large datasets with high dimensionality because it reduces the cardinality of searchable vectors. This approach is superior in speed at the cost of a slight reduction in accuracy. 
+---
+
+# Build k-Nearest Neighbor (k-NN) Similarity Search Engine with OpenSearch
```
Please remove this title  - it appears as duplicative

kolchfa-aws replied:
Done

krisfreedain commented on _posts/2023-01-09-knn-similarity-refresh.markdown:
```diff
@@ -0,0 +1,215 @@
+- laijiang
+- csw
+- jmazane
+date: 2023-01-09 00:00:00 -0700
```
I don't believe we typically update the date here - please confirm @pajuric - and please set this back to the original blog date. This isn't a new blog, it's a refresh.

pajuric replied:
Correct - we leave the original publish date for the blog, which was Mon, Apr 06, 2020.

kolchfa-aws replied:
Put the original date, but named the blog today's date.

krisfreedain reviewed (changes requested):
@kolchfa-aws  a couple changes needed. CC @pajuric

krisfreedain commented:
Just noticed this - curious why the filename was also changed to today's date - this will end up making it a duplicate of the 2020-04-06 post when it should be a refresh of that post/file.

kolchfa-aws commented:
@krisfreedain I changed the file name to the original. I also changed the front matter of the file - could you look it over and see if it's OK, or does it need to be exactly like the original? Thanks.

pajuric commented:
Checking status on this. Do we need to redo the PR to get this one moving?

krisfreedain reviewed (approved):
thank you @kolchfa-aws
